api/src/services/listing-csv-export.service.ts

```diff
@@ -222,19 +258,30 @@ export class ListingCsvExporterService implements CsvExporterServiceInterface {
     });
   }
 
-  async createUnitCsv(filename: string, listings: Listing[]): Promise<void> {
-    const csvHeaders = this.getUnitCsvHeaders();
-    // flatten those listings
-    const units = listings.flatMap((listing) =>
-      listing.units.map((unit) => ({
-        listing: {
-          id: listing.id,
-          name: listing.name,
-        },
-        unit,
-      })),
-    );
-    // TODO: the below is essentially the same as above in this.createCsv
+  async createUnitCsv(
+    filename: string,
+    listings: Listing[],
+    enableUnitGroups?: boolean,
+  ): Promise<void> {
+    const csvHeaders = enableUnitGroups
+      ? this.getUnitGroupCsvHeaders()
+      : this.getUnitCsvHeaders();
```

Comment:
Issue: This doesn't work for the case that both "unit group" jurisdictions as well as "unit" jurisdictions exist in partners. It will only output the unit group version. Instead in that case the zip should have both the `unit.csv` and `unitGroups.csv`